package.json

```diff
@@ -47,7 +47,7 @@
     "eslint": "8.42.0",
     "eslint-config-prettier": "8.8.0",
     "eslint-plugin-prettier": "4.2.1",
-    "got": "13.0.0",
+    "got": "11.8.6",
```

Comment:
Why did you lower the version?

Reply:
Downgraded got lib because it [requires esm module to run](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#pure-esm-package).

Reply:
One day I'll start reading the description